### PR TITLE
Changes for release 2.0 only

### PR DESCRIPTION
**Pull request by weiwangmeta**
Fixes #ISSUE_NUMBER

### CODE REVIEW

**Comment by atalman** [comment]
This change required: https://github.com/pytorch/pytorch/commit/f65ac2ad20f8eab95a0d0fb93c3550afc2b86d91

File: .github/workflows/_binary-build-linux.yml [hunk 1]
```diff
@@ -158,7 +157,6 @@ jobs:
       - name: Checkout pytorch/builder to builder dir
         uses: malfet/checkout@silent-checkout
         with:
-          ref: main
           submodules: recursive
```
Comment (atalman) [inline .github/workflows/_binary-build-linux.yml:160]: need to add ref: release/2.0

**Comment by atalman** [comment]
This change is needed: https://github.com/pytorch/pytorch/pull/90506/files

File: .github/workflows/_binary-build-linux.yml (hunk 1 above)
Comment (weiwangmeta) [reply .github/workflows/_binary-build-linux.yml:160]: Yes, I will add it.

**Review by atalman — APPROVED** [review]
LGTM

**Pull request merged**